### PR TITLE
React query api client generation

### PR DESCRIPTION
**Pull request by artursudnik**
This is a prototype of the code generation during a build. 

To be discussed:
- destination folder
- how to automatically import/export all modules generated
- how to avoid redefining mutator, providers, anything that is going to be shared between autogenerated clients

### CODE REVIEW

**Comment by artursudnik** [comment]
Decisions made:
- destination folder: /packages/api-clients/react-query
- generating index.ts with a script after client .ts files are generated
- shared mutator for all clients

Still to be solved:
- setting backend baseUrl

**Comment by artursudnik** [comment]
This is a prototype for exchange. If it works I will reproduce this pattern for all other clients.
Every client is going to have its own mutator instance. To be solved later if it is an issue.

File: packages/api-clients/react-query/exchange/src/mutator/customInstance.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Axios, { AxiosRequestConfig } from 'axios';
+
+export const AXIOS_INSTANCE = Axios.create({ baseURL: '' });
```
Comment (alexworker23) [inline packages/api-clients/react-query/exchange/src/mutator/customInstance.ts]: I think here as a temporary solution we should add `process.env.BACKEND_URL+process.env.BACKEND_PORT` or new single line `BACKEND_URL` env

Comment (artursudnik) [reply packages/api-clients/react-query/exchange/src/mutator/customInstance.ts]: I agree. I can imagine a situation when someone needs to expose backend API under some path. It is not possible to specify it now with these two variables. But, I'd introduce this change separately.

**Pull request merged**